server/policies/subscription.ts

```diff
@@ -0,0 +1,109 @@
+import { Subscription, User, Document } from "@server/models";
+import { allow, _cannot as cannot } from "./cancan";
+
+// Is `user` allowed to subscribe to `document`?
+allow(User, "createSubscription", Document, (user, document) => {
+  // Sanity check.
+  if (!document) {
+    return false;
+  }
+
+  // REVIEW: Are viewers allowed to subscribe?
+  if (user.isViewer) {
+    return false;
+  }
```

Comment:
Similar to comment above

https://github.com/outline/outline/pull/3834/files#r927311996